app/boards/shields/tidbit/tidbit.dtsi

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+	kscan0: kscan {
+		compatible = "zmk,kscan-gpio-matrix";
+		label = "KSCAN";
+
+		diode-direction = "row2col";
+
+		row-gpios
+			= <&pro_micro_d 15 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_d 7 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_d 6 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_d 5 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_d 4 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			;
+
+        col-gpios
+			= <&pro_micro_a 3 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_a 2 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_a 1 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro_a 0 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			;
+
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,matrix-transform";
+		columns = <4>;
+		rows = <5>;
+
+		map = <
+                    RC(0,1) RC(0,2) RC(0,3)
+            RC(1,0) RC(1,1) RC(1,2) RC(1,3)
+            RC(2,0) RC(2,1) RC(2,2) RC(2,3)
+            RC(3,0) RC(3,1) RC(3,2) RC(3,3)
+            RC(4,0) RC(4,1) RC(4,2) RC(4,3)
+		>;
+	};
+
+	encoder_1_top_row: encoder_1_top_row {
+		compatible = "alps,ec11";
+		label = "Top Row Encoder";
+		a-gpios = <&pro_micro_d 16 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		b-gpios = <&pro_micro_d 14 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		resolution = <4>;
+		status = "disabled";
+	};
+
+	encoder_1: encoder_1 {
+		compatible = "alps,ec11";
+		label = "Encoder 1";
+		a-gpios = <&pro_micro_d 14 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		b-gpios = <&pro_micro_d 16 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		resolution = <4>;
+		status = "disabled";
+	};
+
+	encoder_2: encoder_2 {
+		compatible = "alps,ec11";
+		label = "Encoder 2";
+		a-gpios = <&pro_micro_d 8 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		b-gpios = <&pro_micro_d 9 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		resolution = <4>;
+		status = "disabled";
+	};
+
+	encoder_3: encoder_3 {
+		compatible = "alps,ec11";
+		label = "Encoder 3";
+		a-gpios = <&pro_micro_d 3 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		b-gpios = <&pro_micro_d 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		resolution = <4>;
+		status = "disabled";
+	};
+
+	encoder_4: encoder_4 {
+		compatible = "alps,ec11";
+		label = "Encoder 4";
+		a-gpios = <&pro_micro_d 1 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		b-gpios = <&pro_micro_d 0 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+		resolution = <4>;
+		status = "disabled";
+	};
+
+    chosen {
+		zmk,kscan = &kscan0;
+		zmk,matrix_transform = &default_transform;
+	};
+};
+
+&pro_micro_i2c {
+	status = "disabled";
```

Comment:
Any reason to have this disabled here if you're enabling them in both versions of this shield?

Reply:
Flipping to have the main tidbit dtsi to have "okay" instead of disabled and removing the overrides from the keymaps.